src/webapi/xml_http_request.rs

```diff
@@ -78,6 +113,14 @@ impl XmlHttpRequest {
         }
     }
 
+    /// Returns the object representing the response
+    ///
+    ///[(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/response)
+    // https://xhr.spec.whatwg.org/#ref-for-dom-xmlhttprequest-response
+    pub fn response(&self) -> Result< Value, TODO > {
```

Comment:
1. `response` -> `raw_response` since it returns a raw `Value`; I'd like to reserve the `response` name for a more Rust-y method which returns an enum instead of a `Value`.

2. You return a `Result` here; can this actually throw an exception? [It doesn't look like it can throw to me from the specs](https://xhr.spec.whatwg.org/#dom-xmlhttprequest-response), so it should return `Option` instead.

Reply:
Question: Rather than Option, should it not just return a Value?